packages/experience-builder-sdk/src/index.ts

```diff
@@ -6,6 +6,7 @@ export type {
   ComponentDefinition,
   ComponentRegistration,
   Experience,
+  ExternalSDKMode,
```

Comment:
we might need to also export `EntityStore` type that is taken from the `@contentful/visual-sdk`

Reply:
added this one as well